t/15-assets.t

```diff
@@ -331,6 +331,35 @@ subtest 'check for missing assets' => sub {
         my $job_with_2_assets = $schema->resultset('Jobs')->create_from_settings(\%settings);
         is_deeply($job_with_2_assets->missing_assets, [], 'empty asset not considered so no asset missing');
     };
+    subtest 'private assets are considered' => sub {
+        $settings{HDD_1} = 'disk_from_parent';
+        my $parent_job        = $schema->resultset('Jobs')->create_from_settings(\%settings);
```

Comment:
If you'd like you could get rid of all the repetitive `$schema->resultset('Jobs')` calls in this file (preferably in a separate commit within this PR). So you'd declare `my $jobs = $schema->resultset('Jobs');` somewhere at the top and use that everywhere.